### PR TITLE
Fix `internal/log/logger` to match the test template

### PR DESCRIPTION
**Pull request by hlts2**
Signed-off-by: hlts2 <hiroto.funakoshi.hiroto@gmail.com>

### Description:

I fixed `internal/log/logger` to match the test template.

### Environment:

- Golang Version: 1.14
- Docker Version: 19.03.5
- Kubernetes Version: 1.17.3
- NGT Version: 1.9.1

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [x] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [x] I have added tests and benchmarks to cover my changes.
- [x] I have ensured all new and existing tests passed.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by kevindiu** [comment]
LGTM

**Comment by vankichi** [comment]
@kpango @rinx 
Could you please review, if you have time?

File: internal/log/logger/type_test.go [hunk 1]
```diff
@@ -23,200 +24,243 @@ import (
...
 	}
 
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			got := tt.logType.String()
-			if got != tt.want {
-				t.Errorf("not equals. want: %v, but got: %v", tt.want, got)
+	for _, test := range tests {
+		t.Run(test.name, func(tt *testing.T) {
+			defer goleak.VerifyNone(t)
```
Comment (rinx) [inline internal/log/logger/type_test.go:97]: I don't know but ... is it correct to use `t` here?
i think hlts said `tt` is the correct one.

File: internal/log/logger/type_test.go [hunk 2]
```diff
@@ -23,200 +24,243 @@ import (
...
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
-	}
-
 	for _, test := range tests {
 		t.Run(test.name, func(tt *testing.T) {
 			defer goleak.VerifyNone(t)
```
Comment (rinx) [inline internal/log/logger/type_test.go:252]: same.

File: internal/log/logger/type_test.go (hunk 1 above)
Comment (hlts2) [reply internal/log/logger/type_test.go:97]: Thank you for your review. 
Sorry. There was a correction omission:pray:

**Pull request merged**